packages/e2e-tests/plugins/block-bindings/index.js

```diff
@@ -0,0 +1,50 @@
+const { unlock } =
+	wp.privateApis.__dangerousOptInToUnstableAPIsOnlyForCoreModules(
+		'I acknowledge private features are not for use in themes or plugins and doing so will break in the next version of WordPress.',
+		'@wordpress/blocks'
+	);
+
+const { registerBlockBindingsSource } = unlock( wp.blocks.privateApis );
+const { fieldsList } = window.testingBindings || {};
+
+const getValues = ( { bindings } ) => {
+	const newValues = {};
+	for ( const [ attributeName, source ] of Object.entries( bindings ) ) {
+		newValues[ attributeName ] = fieldsList[ source.args.key ]?.value;
+	}
+	return newValues;
+};
+const setValues = () => {
+	// DO NOTHING
```

Comment:
Maybe we should set an example of setValues, in case people use tests as examples of registering sources like you mention in the PR.

```
it seems it reads better the testing for the client registration APIs, which could help after making them public.
```

Reply:
I added an example in this commit: https://github.com/WordPress/gutenberg/pull/65526/commits/6ed6f43bd5a3cc56f0764dc7179198641a1806be